cli_tools_e2e_test/common/utils/utils.go

```diff
@@ -270,3 +270,32 @@ func ContainsSubString(strs []string, s string) bool {
 	}
 	return false
 }
+
+// ContainsAll checks whether all given strings exists
```

Comment:
it's not clear from the doc what is actually checked? if all from arr are in subarr or vice versa?

Reply:
modified